mentoring/public/js/questionnaire.js

```diff
@@ -99,14 +109,17 @@ function MCQBlock(runtime, element) {
                 var choiceTipsCloseDOM;
 
                 if (result.status === "correct" && choiceInputDOM.val() === result.submission) {
+                    choiceDOM.addClass('correct');
```

Comment:
@singingwolfboy Yes, the `correct` and `incorrect` classes are a toggle, but the classes are cleared by the `clearResult` method (https://github.com/open-craft/xblock-mentoring/blob/mtyaka/tip-styles/mentoring/public/js/questionnaire.js#L62-L68) and are already removed before reaching this part of the code.